src/drivers/usb/class/usb_mass_storage.c

```diff
@@ -177,7 +177,7 @@ static struct usb_device_id usb_ms_id_table[] = {
 	{ },
 };
 
-struct usb_driver usb_driver_ms = {
+static struct usb_driver usb_driver_ms = {
```

Comment:
const?

Reply:
Your suggestion is pretty good, but because of not very good usb driver registration if I add `const`, then I have to change:

```
usb_driver_register(&usb_driver_ms);
```
to 
```
usb_driver_register((struct usb_driver *) &usb_driver_ms);
```

to discard `const` qualifier. That's why I added only `static`.

Reply:
I created an issue with suggestion to make USB drivers statically register - https://github.com/embox/embox/issues/1948

Reply:
Thanks